src/utils/utils.ts

```diff
@@ -259,6 +261,9 @@ export const getWETHAddress = (chain: Chain) => {
     case Chain.B3:
     case Chain.B3Sepolia:
       return "0x4200000000000000000000000000000000000006";
+    case Chain.BeraChain:
+      // https://beratrail.io/token/0x2F6F07CDcf3588944Bf4C42aC74ff24bF56e7590
+      return "0x2f6f07cdcf3588944bf4c42ac74ff24bf56e7590";
```

Comment:
Bera also has WBERA address, should we use wbera instead?

Reply:
should be WETH

Reply:
actually my bad it should be WBERA since that is what we will accept for offers on OS

Reply:
yup this is weth